test/integration/fixtures/wordpress-out.html

```diff
@@ -13,3 +13,15 @@ <h3>More tag</h3>
 <!-- wp:more -->
 <!--more-->
 <!-- /wp:more -->
+
+<!-- wp:heading -->
+<h3>Shortcode</h3>
+<!-- /wp:heading -->
+
+<!-- wp:gallery {"columns":3,"linkTo":"attachment"} -->
+<ul class="wp-block-gallery alignnone columns-3 is-cropped">
+	<li class="blocks-gallery-item">
+		<figure><img data-id="1" /></figure>
```

Comment:
A bit unfortunate that without the editor session to request and populate these attributes, we're effectively mangling the original content and destroying the images which would otherwise be shown by running it through the parser. Not really sure we have an option here outside of allowing transforms to be asynchronous.